src/solver/optim-model-filler/include/antares/solver/optim-model-filler/TimeDependentLinearExpression.h

```diff
@@ -59,9 +59,12 @@ class TimeDependentLinearExpression
     TimeDependentLinearExpression operator-() const;
     TimeDependentLinearExpression shiftLinearExpressions(int shiftValue) const;
```

Comment:
`TimeDependentLinearExpression shiftLinearExpressions(int shiftValue) const;`
is called internally to class **TimeDependentLinearExpression**, and nowhere else (apart from unit tests).
So it should be **private** (a good practice is to keep classes interfaces as smaller as possible).
Unit testing each method of a class leads to have only public methods, which is bad practice.
If we're worried about what to do with unit tests that call this method from outside a **TimeDependentLinearExpression**, we should remember that we should test behaviors, not methods.
We should make small unit tests calling only a small amount of public methods.
Here, we should unit test **timeSumLinearExpressions** instead (the only method calling **shiftLinearExpressions**).

Reply:
it"s called from ReadLinearExpressionVisitor::visit(const TimeShiftNode* node) && i do test behavior